plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/lifecycle-mapping-metadata.xml

```diff
@@ -1,19 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<lifecycleMappingMetadata>
```

Comment:
Ah great, versions-maven-plugin 2.3 added their own lifecycle-mapping.  As it should be.